goalie/go_mesh_seq.py

```diff
@@ -174,7 +174,11 @@ def indicate_errors(
             for f, fs_e in enriched_spaces.items():
                 u[f] = Function(fs_e)
                 u_[f] = Function(fs_e)
-                mapping[f] = (u[f], u_[f])
+                mapping[f] = (
+                    (u[f], u_[f])
+                    if enriched_mesh_seq.field_types[f] == "unsteady"
+                    else u[f]
+                )
```

Comment:
In the steady case I guess we don't need to define `u_` at all?

Reply:
I was thinking about this, yeah. We do not need to define `u_` only if the problem is steady, since "steady" fields in unsteady problems still have lagged solutions (as we discussed in #153). But I think your existing code is actually more elegant, as it keeps the rest of `indicate_errors` very tidy. Yes, there is unnecessary effort for steady problems where we add `u` and `u_` (which are equal) and divide by two, but this is minor inefficiency since the problem is steady and we only do this once :)

But I'm happy to change this if you think it's better!

Reply:
Fair point, thanks.